pkg/proxy/ipvs/proxier_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/vishvananda/netlink"
```

Comment:
this seems the reasons that verify check breaks, since it doesn't compile in windows

ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/proxy/ipvs/proxier_test.go:5429:9: undeclared name: netlinkHandle
ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/vishvananda/netlink/xfrm.go:13:36: IPPROTO_ROUTING not declared by package unix
ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/vishvananda/netlink/xfrm.go:14:36: IPPROTO_ESP not declared by package unix